garden_ai/app/model.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+from typing import Optional, List
+
+from garden_ai.client import GardenClient
+
+import typer
+import rich
+
+model_app = typer.Typer(name="model", no_args_is_help=True)
+
+
+@model_app.callback()
+def pipeline():
```

Comment:
afaik it's ignored everywhere so this matters ~not at all, but could we rename the just-for-docstring function to `model` here

Reply:
d'oh. Yeah this is some shameful copypasta